x-pack/docs/en/security/authorization/privileges.asciidoc

```diff
@@ -20,6 +20,25 @@ See <<delete-async-search,delete async search>> API for more informations.
 Privileges to create snapshots for existing repositories. Can also list and view
 details on existing repositories and snapshots.
 
+// TODO: fix the link to new page of API key based remote clusters
+`cross_cluster_replication`::
+beta:[]
+Privileges to connect to remote clusters configured with the API key based model
+for cross-cluster replication.
+
+NOTE: This privilege should _not_ be directly granted. It is used internally by
+<<security-api-create-cross-cluster-api-key>> and <<security-api-update-cross-cluster-api-key>>
+to manage cross-cluster API keys.
```

Comment:
Writing this makes me think that `cross_cluster_replication` and `cross_cluster_search` cluster privileges could have been suffixed with `_internal` similar to the `cross_cluster_replication_internal` privilege. Is it worth the effort?

Reply:
Good point, but end-users won't directly interact with these privileges, so I'd say it's fine to leave as is.